webapp/components/signup_user_complete.jsx

```diff
@@ -486,7 +504,7 @@ class SignupUserComplete extends React.Component {
                 <div className='col-sm-12'>
                     <div className='signup-team__container padding--less'>
                         <div>
-                            <form>
+                            <div>
```

Comment:
If it's a form it creates a React warning since LoginLdap has a form (can't have a form within a form). I'll add manual checking for the enter key